packages/client/src/router/index.js

```diff
@@ -168,6 +167,7 @@ router.beforeEach((to, from, next) => {
   } else if (to.name === 'not-found'
     || (to.meta.requiresMyProfileEnabled && !myProfileEnabled())
     || (to.meta.requiresNewTerminologyEnabled && !newTerminologyEnabled())
+    || (to.meta.requiresNewGrantsDetailPageEnabled && !newGrantsDetailPageEnabled())
```

Comment:
@replicantSocks does adding this eliminate the need to check the feature flag in some of the other spots?